star-rx72n-firmware/tests/test_gpio_hal.c

```
@@ -119,20 +134,21 @@ void test_gpio_set_output_error_injection(void)
  */
 void test_gpio_set_output_call_history(void)
 {
-  (void)gpio_set_output(k_gpio_pb2);
-  (void)gpio_set_output(k_gpio_pa0);
+  rx_err_t err = gpio_set_output(k_rx_pb_2);
+  (void)gpio_set_output(k_rx_pa_0);
 
+  TEST_ASSERT_EQUAL(k_rx_ok, err);
   TEST_ASSERT_EQUAL(2, mock_gpio_get_call_count());
 
   const mock_gpio_call_t* call0 = mock_gpio_get_call(0);
```

Clang (14.0.6)

[warning] variable 'call0' is not initialized

(cppcoreguidelines-init-variables)